templates/node/src/client.ts.twig

```diff
@@ -1,8 +1,11 @@
 import { fetch, FormData, File } from 'node-fetch-native-with-agent';
 import { createAgent } from 'node-fetch-native-with-agent/agent';
 import { Models } from './models';
+import { Payload } from './Payload';
```

Comment:
I think the file path should `./payload` lowercase as of the copy action [here](https://github.com/appwrite/sdk-generator/pull/964/files#diff-947d6880be0e5cb7ad5aaef122edb9500aad7a480bcc6eef441a1478f5ea0a94R167)